src/In.ProjectEKA.HipService/Gateway/GatewayPathConstants.cs

```diff
@@ -0,0 +1,9 @@
+namespace In.ProjectEKA.HipService.Gateway
+{
+    public class GatewayPathConstants
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5570468)